clusterman/draining/queue.py

```diff
@@ -248,7 +251,20 @@ def process_drain_queue(
                     self.submit_host_for_termination(host_to_process)
             elif host_to_process.scheduler == "kubernetes":
                 logger.info(f"Kubernetes host to drain and submit for termination: {host_to_process}")
-                self.submit_host_for_termination(host_to_process, delay=0)
+                try:
+                    if kube_operator_client:
+                        k8s_drain(
+                            kube_operator_client, host_to_process.hostname,
```

Comment:
We can't drain k8s node with host_to_process.hostname. We need agent.agent_id (node name)
P.S. I know this PR is stale, just in case..